src/core/server/saved_objects/routes/copy.ts

```diff
@@ -1,12 +1,19 @@
 /*
- * Copyright OpenSearch Contributors
  * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
```

Comment:
The code in copy.ts and copy.test.ts are old, please use the code in the `workspace-pr-integr` branch.

Reply:
Have reverted these codes